.github/workflows/ocwm-issue-collector.yml

```
@@ -111,6 +113,8 @@ jobs:
                     let json_text = JSON.stringify(body.substring(0, startIndex - 1) + `\n| ${url} - ${title} | ${author} |\r\n` + body.substring(startIndex, body.length));
                     body = JSON.parse(json_text);
                     changesFlag = true;
+                    // Remove the 'agenda' label from the added discussion
+                    await mygithub.request(`DELETE /repos/${process.env.OWNER}/${repositories[r]}/discussions/${discussions[i].node.id}/labels/${appendLabel}`);
```

```suggestion
                    await mygithub.request(`DELETE /repos/${process.env.OWNER}/${repositories[r]}/discussions/${discussions[i].number}/labels/${appendLabel}`);
```

Why not using the property name like you did for issues?

---

You're correct; it's more consistent and clearer to use the property name directly instead of relying on the index